src/modules/audio/audio.c

```diff
@@ -30,21 +36,18 @@ struct Source {
   bool playing;
   bool looping;
   bool spatial;
+  float pose[16];
+  int output_channel_count;
```

Comment:
Style: this struct field (and capture_ringbuffer) should be camelCase

Reply:
Also, this value depends entirely on `spatial` and can be computed from it efficiently.  Storing it separately doesn't feel necessary and there is a risk that they accidentally drift (e.g. we add the ability to change spatialness and now have to remember to keep the output channel count in sync).  This is pretty minor though.  It could also be the case that in the future we *do* want to separate them for a valid reason.

Reply:
I made it a separate computed value to be explicit. I don't usually like duplicate state but this felt better than hard-coding the assumption between spatial and channels into parts of the code. Could make a little static inline helper function to make it explicit in a single place (should have same performance)